packages/typespec-ts/package.json

```diff
@@ -62,7 +62,7 @@
     "tslib": "^2.3.1",
     "@azure-tools/rlc-common": "^0.12.0",
     "ts-morph": "^15.1.0",
-    "@azure-tools/typespec-client-generator-core": "~0.29.0",
+    "@azure-tools/typespec-client-generator-core": "~0.30.0-dev.9",
```

Comment:
why not wait until this is released ?

Reply:
I take this change because it is a blocker for MAVD, but I think it's okay to hold this pr if TCGC could release quickly.

Reply:
Let's block this first.